packages/suite/src/config/suite/yup.ts

```diff
@@ -0,0 +1,22 @@
+import { isHex as isHexUtil } from '@suite/utils/wallet/ethUtils';
+import { isAscii as isAsciiUtil } from '@trezor/utils';
+import * as yup from 'yup';
+
+yup.setLocale({
+    string: {
+        max: 'TR_EXCEEDS_MAX',
```

Comment:
nit: do we have type for message keys? would be really nice to have this keys typed to ensure that they are really defined

Reply:
Unfortunately no, there is no build in ability to type the keys. Not sure how to override the `Message` type in yup.